Source/WebKit/WebProcess/FullScreen/WebFullScreenManager.cpp

```diff
@@ -37,8 +37,8 @@
 #include "WebPage.h"
```

Comment:
What about WebFullScreenManager ? that seems out of place too

Reply:
I think this is fine as `WebFullScreenManager`, we could perhaps make that `FullscreenManager`, but there's little benefit to renaming it.